docs/spec/relayer/Relayer.tla

```diff
@@ -118,8 +38,10 @@ IsChannelOpen(chainID) == Env!IsChannelOpen(chainID)
 \* Some client updates might trigger an update of the height that 
 \* the relayer stores for srcChainID
 LightClientUpdates(srcChainID, dstChainID, relayer) ==
-    LET srcChainHeight == GetLatestHeight(srcChainID) IN    
-    LET srcClientHeight == GetMaxCounterpartyClientHeight(dstChainID) IN
+    LET srcChain == GetChainByID(srcChainID) IN
+    LET dstChain == GetChainByID(dstChainID) IN
+    LET srcChainHeight == GetLatestHeight(srcChain) IN    
```

Comment:
We should probably parameterise datagram generation functions with a height for which we are creating datagrams. If datagrams are created for height h, then the light client updates are created for the height h+1 as root hash for datagrams created at height h is stored in the block h+1.

Reply:
Would be a nice improvement to parametrize datagram generation this way!